ankihub/gui/web/templates.py

```diff
@@ -47,10 +47,11 @@ def get_reviewer_buttons_js(theme: str, enabled_buttons: List[str]) -> str:
         {
             "THEME": theme,
             "ENABLED_BUTTONS": ",".join(enabled_buttons),
+            # TODO
+            "IS_PREMIUM": str(True),
```

Comment:
I assume we don't have an API endpoint to check if the user is premium, so wee need to add one?

Reply:
The endpoint already exists, but we don't send the plan data through it. We will need to include the premium info on it!

There is the `/users/me` which retrieves information about the user but we need to include the premium info.
but we could create a new function on the `AnkihubClient` similar to the `owned_deck_ids` but to store the is_premium info.